src/libraries/System.Diagnostics.PerformanceCounter/src/System/Diagnostics/PerformanceCounter.cs

```diff
@@ -79,7 +79,7 @@ public PerformanceCounter(string categoryName, string counterName, string instan
             GC.SuppressFinalize(this);
         }
 
-        internal PerformanceCounter(string categoryName, string counterName, string instanceName, string machineName, bool skipInit)
+        internal PerformanceCounter(string categoryName, string counterName, string instanceName, string machineName, bool _)
```

Comment:
This parameter name was self-documenting about how this constructor differs from public constructor above.  I think it'd be better to change the previous constructor to be like:
```C#
public PerformanceCounter(string categoryName, string counterName, string instanceName, string machineName) :
    this(categoryName, counterName, instanceName, machineName, skipInit: false)
{
}
```
and then rather than changing this parameter name, change the implementation below to do:
```C#
if (skipInit)
{
    _initialized = true;
}
else
{
    Initialize();
}
```

Reply:
ok